lib/utils/defer.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * Promisify a function of the form (value, options, cb)
+ * @param  {Function} asyncFunc The function
+ * @return {Function} 					The new function, as a Promise
```

Comment:
I can fix it :smile: It looks awful in github, but works nicely inside an editor.

<img width="591" alt="screen shot 2016-03-13 at 2 11 10 pm" src="https://cloud.githubusercontent.com/assets/5855893/13731424/74400672-e925-11e5-82c2-4f6f02024b87.png">